app/components/shared/PolicyPicker/PolicyPicker.jsx

```diff
@@ -47,15 +52,39 @@ class PolicyPicker extends React.Component {
         )
     }
 
-    reloadPolicyList() {
-        tokenHasCapabilities(['read'], 'sys/policy')
+    reloadPolicyList(type) {
+        let allowed_methods = ['']
+        let http_method = ''
+        let path = ''
+        let params = null
+        switch (type) {
+            case "okta/users":
```

Comment:
As you can see here, I modified `PolicyPicker` quite a bit in order to have it be more of a general `Picker`. I have not gone through and renamed everything in here, I figure I'd do that later on in a different PR since the regression chance is greater.

By doing this, we can utilize group picking for Okta users using the same Picker as whats used for policies.